src/features/auth/use-auth.ts

```diff
@@ -42,7 +42,6 @@ export const useAuth = () => {
           window.location.pathname !== path.signin() // Unless user is already at signin page
         ) {
           window.location.href = path.signin();
-          window.alert("Please signin.");
```

Comment:
Can you leave this code for now to notify a user that the user is redirected because the user is not signed in? Or if there is any purpose, let me know 👍